packages/angular_devkit/build_angular/src/builders/karma/tests/options/aot_spec.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import { execute } from '../../index';
+import { BASE_OPTIONS, KARMA_BUILDER_INFO, describeKarmaBuilder } from '../setup';
+import { BuilderMode } from '../../schema';
+
+describeKarmaBuilder(execute, KARMA_BUILDER_INFO, (harness, setupTarget) => {
+  describe('Option: "aot"', () => {
+    it('enables aot', async () => {
+      await setupTarget(harness);
+
+      await harness.writeFiles({
+        'src/aot.spec.ts': `
+          import { Component } from '@angular/core';
+          
+          describe('Hello', () => {
+            it('should *not* contain jit instructions', () => {
+              @Component({
+                template: 'Hello',
+              })
+              class Hello {}
+
+              expect((Hello as any).ɵcmp.template.toString()).not.toContain('jit');
```

Comment:
This is fragile and sensitive to any structural changes in JIT. It could easily cause false negatives.
What would be a better way to achieve this detection?

Reply:
I think as long as we have a second test case that tests the reverse, it's fine..? That way we can make sure that "jit" appears in one branch and doesn't appear in the other, with `aot: true` vs `aot: false` as the only difference. That seems like an okay state to be in for now.